cmd/veeam-sos-api.go

```diff
@@ -85,16 +85,6 @@ type systemInfo struct {
 		UploadSessions bool `xml:"UploadSessions"`
 		IAMSTS         bool `xml:"IAMSTS"`
 	} `mxl:"ProtocolCapabilities"`
-	APIEndpoints struct {
```

Comment:
You shouldn't be removing this also, if this is optional in the sense it expects them to be empty then it needs to be done differently.

It is kept here for future purposes